app/services/discount_manager.py

```
@@ -0,0 +1,45 @@
+from datetime import datetime, timedelta
+import random
```

Ruff (0.12.2)

`random` imported but unused

Remove unused import: `random`

(F401)